docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -258,8 +258,6 @@ Following are some scenarios where this can happen:
 
 **Configuration**
 
-Fingerprint mode is disabled by default.
-
 ::::{warning}
 Enabling fingerprint mode delays ingesting new files until they grow to at least `offset`+`length` bytes in size, so they can be fingerprinted. Until then these files are ignored.
 ::::
```

Comment:
Should we say that it is enabled by default? The language in the `warning` below the sentence you removed might suggest to the reader that fingerprint mode needs to be enabled.

Reply:
it's done at the very beginning of the [docs](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-filestream):
> The default behaviour is to identify files based on their contents using [fingerprint](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-filestream#filebeat-input-filestream-file-identity-fingerprint) [file_identity](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-filestream#filebeat-input-filestream-file-identity). This solves data duplication caused by inode reuse.